packages/aws-cdk/test/integ/cli-regression-patches/v1.62.0/skip-tests.txt

```diff
@@ -0,0 +1,8 @@
+# This file is empty on purpose. Leave it here as documentation
```

Comment:
You don't need to include this file if you're not going to use it.

Reply:
k